src/PKSim.Core/Services/InteractiveSimulationRunner.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using OSPSuite.Core.Events;
+using OSPSuite.Core.Serialization.SimModel.Services;
+using OSPSuite.Utility.Events;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Services
+{
+   public interface IInteractiveSimulationRunner : IListener<SimulationResultsUpdatedEvent>
```

Comment:
This is the one called from the UI that actually also triggers output selection if required, or update charts. 
CLI or Test do not need to deal with that